Gemfile

```diff
@@ -21,13 +21,15 @@ gem 'puma', '~> 4.1'
 
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.4.2', require: false
-
+gem 'rack-cors'
```

Comment:
This may have been implemented in the wrong location in the file. Will look into this before next PR.